canonical-web-and-design/ubuntu.com
# Updates copy on /security/livepatch

## Done

- Updates copy on /security/livepatch
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/livepatch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy-doc](https://docs.google.com/document/d/1Ofw9bQ4I40gy9hDIjuDMApz6sDK8qSdHI03-oY7xs-A/edit#) and accept changes


## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9758
